Hackspace Owners Manual/HOM/contributing.md

```diff
@@ -0,0 +1,198 @@
+contribute.
+
+Contributions should developed within a branch and submitted as pull requests.
+In order to do this, you will need to follow a few steps.
+
+* Fork the git repository
+
+Visit https://github.com/UKHackspaceFoundation/resources and select 'Fork' in the top right of the page. You will be asked where you'd like to fork the repository to, choose your own account.
+
+* Clone the new repository
+
+In your newly created clone repository you will see a dropdown for 'Clone or
+download'. Drop this down and select the mechanism you usually use to clone
+(ssh is usual for Linux users), copy the text. Use this text to clone the repository. Assuming use of the command line client, this would look like this:
+```
+git clone git@github.com:idnorton/resources.git
+```
+
+You would expect idnorton to be replaced by your own github username.
+
+* Create a feature branch
+
+Once you have a clone of your repository, you should create a feature branch.
+For the command line client this might look like this:
+```
+git checkout -b feature/my-feature-request
+```
+
+Where my-feature-request is replaced by a suitable feature request text.
+
+* Make your changes
+
+Follow the usual process of git add, commit and push to get your required
+changes up to your github repository. If you're not sure how to do this then
+feel free to ask for assistance on [the
+forum](https://forum.hackspace.org.uk/).
+
+Please remember to use sensible commit messages, if you need assistance on how
+those might look then we recommend [How to write a commit
+message](https://chris.beams.io/posts/git-commit/)
+
+* Create a pull request
+
+Once you've made your changes, committed them and pushed them up to your forked repository it's time to create the pull request. In your own fork, click on the button labelled 'New pull request'. This will take you to a new view, check that the following are correct:
+```
+base fork: UKHackspaceFoundation/resources
+base: develop
+```
+In head fork should be your own repository followed by your branch.
+
+You should be able to see all of your changes, carefully review these to make
+sure that only the things you expect to have changed have changed. Next click
+on the 'Create pull request' button and enter a suitable subject line and
+description. When you are ready and happy with your subject line and
+description click on the 'Create pull request' button.
+
+Now you'll have to wait until someone is able to review your request and merge
+it in for you. We'll do that as quickly as we can for you :) If you've seen no
+reply and it's been a week since you submitted your Pull Request, please feel
+free to post on the forum asking for someone to find time to review it.
+
+We don't intentionally ignore people and we love it when people contribute
+help, but we all have day jobs and sometimes might not have enough energy to
+reply to you as quickly as we would like. We appreciate your efforts.
+
+# Release the Manual!
```

Comment:
This is poorly titled. Probably needs changign to Admin or something and we need a section on dealing with pull requests, possibly including suggestions on how to be tactful, encouraging and not piss people off ;)